<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mdiWorker</class>
 <widget class="QWidget" name="mdiWorker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Worker</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Search by:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="8">
    <widget class="QLineEdit" name="editSearch"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>First name:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Address:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>NINO</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5" colspan="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Bank Short Code</string>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Max h/day</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="editFirstName"/>
   </item>
   <item row="3" column="1" rowspan="5" colspan="2">
    <widget class="QTextEdit" name="editAddress"/>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="QLineEdit" name="editNino"/>
   </item>
   <item row="3" column="5" colspan="2">
    <widget class="QLineEdit" name="editBankShortcode"/>
   </item>
   <item row="3" column="7">
    <widget class="QSpinBox" name="editHday"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Second name</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>UTR</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5" colspan="2">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Bank Account</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="editSecondName"/>
   </item>
   <item row="5" column="3" colspan="2">
    <widget class="QLineEdit" name="editUtr"/>
   </item>
   <item row="5" column="5" colspan="2">
    <widget class="QLineEdit" name="editBankAccount"/>
   </item>
   <item row="5" column="7">
    <widget class="QCheckBox" name="editSubcontractor">
     <property name="text">
      <string>Subcontractor</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Date of birth</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Card ID</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QDateEdit" name="editDob"/>
   </item>
   <item row="7" column="3" colspan="2">
    <widget class="QLineEdit" name="editCardId"/>
   </item>
   <item row="8" column="0" colspan="8">
    <widget class="QTableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="buttonClose">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="buttonCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QPushButton" name="buttonNew">
     <property name="text">
      <string>New</string>
     </property>
    </widget>
   </item>
   <item row="9" column="4" colspan="2">
    <widget class="QPushButton" name="buttonEdit">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="9" column="6" colspan="2">
    <widget class="QPushButton" name="buttonDelete">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="7" column="5">
    <widget class="QLineEdit" name="editWages"/>
   </item>
   <item row="7" column="7">
    <widget class="QDateEdit" name="editFromDate">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Current wage/h</string>
     </property>
    </widget>
   </item>
   <item row="6" column="7">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Date wage efective</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
